include/deal.II/grid/reference_cell.h

```diff
@@ -588,7 +588,18 @@ class ReferenceCell
   bool
   contains_point(const Point<dim> &p, const double tolerance = 0) const;
 
-  /*
+  /**
+   * Return the point on the surface of the unit cell closest (in the Euclidean
+   * norm) to @p.
+   *
+   * @note the point returned by this function may, due to roundoff, be slightly
+   * outside the unit cell.
+   */
+  template <int dim>
+  Point<dim>
+  project_to_unit_cell(const Point<dim> &p) const;
```

Comment:
We've moved away from the term "unit cell" and replaced that by "reference cell". Perhaps rename the function?

Reply:
`reference_cell` is redundant here since the class is already named `ReferenceCell` so I went with `closest_point()`.